SimMuon/GEMDigitizer/src/ME0PreRecoGaussianModel.cc

```diff
@@ -77,6 +79,8 @@ for (const auto & hit: simHits)
   double corr=0.;
   double tof=CLHEP::RandGaussQ::shoot(engine, hit.timeOfFlight(), sigma_t);
   int pdgid = hit.particleType();
+  if (ex == 0) ex = error_u;//errors cannot be zero
+  if (ey == 0) ey = error_v;
```

Comment:
Isn't this an error prone operation? Wouldn't it be better to check e.g. ex < 1e-6?